lib/storage-client.js

```diff
@@ -243,6 +243,8 @@ class ChromedriverStorageClient {
       await fs.mv(chromedriverPath, dst, {
         mkdirp: true
       });
+      fs.chmod(dst, 0o755);
+      log.debug(`The permissions for file '${dst}'  have been changed to 755!`);
```

Comment:
extra space after '${dst}', the exclamation mark is redundant

Reply:
Thank you